src/templates/base/2019/base.html

```diff
@@ -99,11 +99,134 @@
+        <option disabled>
+          {{ self.chapter() }} {{ chapter_config.chapter }}: {{title}}
+        </option>
+      {% else %}
+        {% if chapter_lang_exists(lang, year, chapter_config.slug) %}
+          {% if current_title == title %}
+            <option selected value="{{ url_for('chapter', year=year, lang=lang, chapter=chapter_config.slug) }}">
+              {{ self.chapter() }} {{ chapter_config.chapter }}: {{title}}
+            </option>
+          {% else %}
+            <option value="{{ url_for('chapter', year=year, lang=lang, chapter=chapter_config.slug) }}">
+              {{ self.chapter() }} {{ chapter_config.chapter }}: {{title}}
+            </option>
+          {% endif %}
+        {% else %}
+        {% if current_title == title %}
+          <option selected value="{{ url_for('chapter', year=year, lang='en', chapter=chapter_config.slug) }}">
+            {{ self.chapter() }} {{ chapter_config.chapter }}: {{title}}
+          </option>
+        {% else %}
+          <option value="{{ url_for('chapter', year=year, lang='en', chapter=chapter_config.slug) }}">
+            {{ self.chapter() }} {{ chapter_config.chapter }}: {{title}}
+          </option>
+        {% endif %}
+        {% endif %}
+      {% endif %}
+    {% endfor %}
+    {% endfor %}
+
+    <option value="{{ url_for('methodology', year=year, lang=lang) }}">
+      {{ self.methodology_title() }}
+    </option>
+    <option value="{{ url_for('contributors', year=year, lang=lang) }}">
+      {{ self.contributors_title() }}
+    </option>
+    <option value="/static/pdfs/web_almanac_{{ year }}_{{ lang }}.pdf">
+      {{ self.ebook_title() }}
+    </option>
+  </select>
+</div>
+{% endmacro %}
+
+{% macro styled_table_of_contents_dropdown(switcher_name) %}
+{% set current_title = metadata.get('title') if metadata else None %}
+<div class="nav-dropdown {{switcher_name}} table-of-contents">
+  <button class="nav-dropdown-btn enable-script" aria-expanded="false" aria-label="{{ self.table_of_contents_title() }}" >
+    <a class="js-enable-script-remove-tag" href="{{ url_for('table_of_contents', year=year, lang=lang) }}">{{ self.table_of_contents_title() }}</a>
+  </button>
+  <ul class="nav-dropdown-list hidden {{switcher_name}}-list">
+    
+    <li class="nav-dropdown-list__chapter foreword">
+      <a href="{{ url_for('table_of_contents', year=year, lang=lang) }}">{{ self.foreword_title() }}</a>
+    </li>
+
+    {% for part_config in config.outline %}
+    <li class="nav-dropdown-list__part">
+      {{ self.part() }} {{ localizedPartTitles[part_config.part] if localizedPartTitles[part_config.part]|length else chapter_config.title }}
+    </li>
+    {% for chapter_config in part_config.chapters %}
+    {% set title = localizedChapterTitles[chapter_config.slug] if localizedChapterTitles[chapter_config.slug]|length else chapter_config.title %}
+    <li class="nav-dropdown-list__chapter {% if current_title == title %}active{% endif %}">
```

Comment:
Suggesting the newly added `nav-dropdown-list__current` class here here to grey it out (though could also be renamed active if you prefer):

```suggestion
    <li class="nav-dropdown-list__chapter {% if current_title == title %}nav-dropdown-list__current{% endif %}">
```

Reply:
i fixed.